var/spack/repos/builtin/packages/ruby/package.py

```diff
@@ -4,9 +4,14 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 import re
+import sys
 
+from spack import *
 
-class Ruby(AutotoolsPackage):
+is_windows = sys.platform == 'win32'
+
+
+class Ruby(Package):
```

Comment:
Thanks for the feedback. At this point in the process, we really need to get this branch merged and do clean up work after that. We have several useful packages working on windows and CI for windows that makes sure those keep working. The current goals of the PR are to not break any Linux/Mac OS tests, and to add CI on windows. So that future changes on Linux/Mac are not breaking what is already working on Windows. We should be landing this branch soon (days not week). There are a list of things that will be fixed after this branch is merged. However, right now making changes on this branch is very painful as it involves rebasing and playing catchup with changes made for linux/macos. There is a long term commitment to continue this effort after this initial merge. However, it will be much easier to work on this once this branch gets merged and the changes are smaller going forward with CI protecting the Windows code going forward.